book/http.md

```diff
@@ -835,6 +835,21 @@ not recommended. It's very difficult to write a custom TLS
 implementation that is not only correct but secure.
 :::
 
+At this point you should be able to run your program on any web plage.
+Here is what it should output for [this example][example-simple]:
+
+[example-simple]: examples/example1-simple.html
```

Comment:
I'm OK with this but why not use `example.com`? It's been a running example in most of the chapter.

Reply:
I tried that, but it spams out a lot of text from the style sheet. Rather than have a wall of text that covers two pages, or omitting part of the actual output, I opted for a simpler web page.